internal/controller/clusterlogging/clusterlogging_controller.go

```diff
@@ -67,6 +67,11 @@ func (r *ReconcileClusterLogging) Reconcile(ctx context.Context, request ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	// skip reconcile process if Unmanaged state
+	if instance.Spec.ManagementState == loggingv1.ManagementStateUnmanaged {
+		return ctrl.Result{}, nil
```

Comment:
Can we add a reasoncode to the Ready condition and set it to false

Reply:
@jcantrill Why set it to `false`?

Reply:
Something like this:
```
cond := loggingv1.NewCondition(loggingv1.ConditionReady, corev1.ConditionFalse, 
			  loggingv1.ReasonUnmanagedState, "Skip reconcile if managementState is Unmanaged")
```

Reply:
Because the condition is "ready" and the implication is we are not ready because we are unmanaged, so the condition does not apply